captcha-validation-service/pom.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.mosip.esignet</groupId>
+        <artifactId>esignet-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+	<groupId>io.mosip.esignet.captcha</groupId>
+	<artifactId>captcha-validation-service</artifactId>
+
+	<name>captcha-validation-service</name>
+	<description>A pre-registration service to generate and validate captcha</description>
+	<version>1.2.1-SNAPSHOT</version>
+
+	<properties>
+		<java.version>11</java.version>
+		<maven.jar.plugin.version>3.0.2</maven.jar.plugin.version>
+		<io.micrometer.prometheus.version>1.4.2</io.micrometer.prometheus.version>
+		<pre.registration.core.version>1.2.1-SNAPSHOT</pre.registration.core.version>
+		<spring-cloud-config.version>2.0.0.RELEASE</spring-cloud-config.version>
+		<kernel.logger.logback.version>1.2.1-SNAPSHOT</kernel.logger.logback.version>
+		<kernel.version>1.2.1-SNAPSHOT</kernel.version>	
+	</properties>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-dependencies</artifactId>
+				<version>${spring.boot.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.projectlombok</groupId>
+				<artifactId>lombok</artifactId>
+				<version>${lombok.version}</version>
+				<scope>compile</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+	<dependencies>
+	  <dependency>
+			<groupId>io.mosip.kernel</groupId>
```

Comment:
Why is kernel-core used?

Reply:
dateutils of kernel-core is being used in the code anusha